pkg/util/converter/jsonitere/jsonitere.go

```diff
@@ -1,66 +0,0 @@
-// Package jsonitere wraps json-iterator/go's Iterator methods with error returns
-// so linting can catch unchecked errors.
-// The underlying iterator's Error property is returned and not reset.
-// See json-iterator/go for method documentation and additional methods that
-// can be added to this library.
-package jsonitere
```

Comment:
@itsmylife i missed this before approving, but it looks like this is still used elsewhere in the repo

Reply:
Yes, it was used here https://github.com/grafana/grafana/pull/82360.
I'll first merge influxdb PR then this one.